datalist-polyfill.js

```diff
@@ -17,7 +17,18 @@
 
 	// in case of that the feature doesn't exist, emulate it's functionality
 	if ( !nativedatalist ) {
-		
+		//Customevent for IE
+		(function () {
```

Comment:
I don't think you need to polyfill the `CustomEvent` type.
You can probably remove this, then down below, you can just use `Event` instead of `CustomEvent`